code/io/timer.cpp

```diff
@@ -104,7 +106,12 @@ std::uint64_t timer_get_nanoseconds()
 {
 	auto time = get_performance_counter();
```

Comment:
So now I have a better idea of what happens: 
In Windows, the value is in 1024 clock cycles. so typically ~2500000 assuming most computers run between 2 and 3 ghz. In Linux, the value on most modern kernels is 1000000000.